packages/r/regression.json

```
@@ -16,13 +16,13 @@
     "analysis"
   ],
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/Tom-Alexander/regression-js.git",
+    "source": "npm",
+    "target": "regression",
```

```suggestion
    "source": "git",
    "target": "git://github.com/Tom-Alexander/regression-js.git",
```
The [git repo still has the correct files with tags](https://github.com/Tom-Alexander/regression-js/tree/2.0.1/dist), so let's not complicate the change by changing where we get files from.